package.json

```diff
@@ -48,6 +48,7 @@
   },
   "dependencies": {
     "debug": "^2.2.0",
+    "electron-editor-context-menu": "^1.1.0",
```

Comment:
Yeh, this was just to get the initial going - I think we'll want our own because will likely want a different context menu depending on what is clicked